### ISSUE TITLE
Modify template to only complain about mission Android SDK when interacting with the android project.

### ISSUE THREAD

**Issue by ajhager** [ajhager/libgdx-sbt-project.g8#65]
If someone only wants to develop for desktop platforms, they shouldn't be bothered to install android or ios sdks.

**Comment by aginiewicz** [comment]
I wonder, have you tested https://github.com/pfn/android-sdk-plugin ? It seems to be gaining momentum recently and has some features we wait for in plugin we use right now, like sbt 0.13 and customizing proguard.

**Comment by ajhager** [comment]
Yeah, I have been keeping a close eye on it. The lack of sbt 0.13 support in our current plugin is really starting to put a sour taste in my mouth. I think it is time see if android-sdk-plugin would be a good fit for us.

**Comment by aginiewicz** [comment]
No matter if we move or not, I've just checked that with current plugin it is impossible. Almost every file in android-plugin imports AndroidPlugin._, which contains settings field checking for SDK path. It means that importing almost anything, even like AndroidProject, causes the check to be forced. I've opened https://github.com/jberkel/android-plugin/issues/199 with proposed solution - if it was only warning not error, it would do.

**Comment by ajhager** [comment]
Thanks for looking into this. I could live with a warning.

**Comment by aginiewicz** [comment]
with new template it still complains about it... I think we need to ask upstream about it.

```
[error] set ANDROID_HOME or run 'android update project -p ...'
```

**Comment by ajhager** [comment]
I can confirm that pfn's workaround does not help. I will see if I can come up with a solution.
